extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeSubscriptionExtractor.java

```diff
@@ -103,7 +103,7 @@ private String readFromInputStream(InputStream inputStream) throws InvalidSource
             }
         } catch (InvalidSourceException e) {
             throw e;
-        } catch (Throwable e) {
+        }   catch (Throwable e) {
```

Comment:
Why this indentation? Plus as you said, you shouldn't catch `Throwable`, and this file isn't even related to your PR

Reply:
No, I was just browsing with vscode and sent it on irc for bad practise. I'll remove it.